spel/minimal-linux.json

```diff
@@ -587,4 +587,4 @@
         "vagrantcloud_token": "{{env `VAGRANTCLOUD_TOKEN`}}",
         "vagrantcloud_username": ""
     }
-}
+}
```

Comment:
This is indicating a new line is missing (recommend installing the editorconfig plugin)